lib/core/utilities.js

```diff
@@ -17,10 +17,11 @@ export const parseQuery = queryString => {
 }
 
 export const encodeQuery = queryObject => {
-  return Object.keys(queryObject)
+  return Object.entries(queryObject)
+    .filter(([key, value]) => value != null)
```

Comment:
Should we ignore all falsy values? ( => !value)

Reply:
just need ignore undefined and null